db/crud.go

```diff
@@ -2247,7 +2247,7 @@ func (db *Database) getChannelsAndAccess(ctx context.Context, doc *Document, bod
 			}
 
 		} else {
-			base.WarnfCtx(ctx, "Sync fn exception: %+v; doc = %s", err, base.UD(body))
+			base.WarnfCtx(ctx, "Sync fn exception: %+v; doc_id = %s, rev_id = %s", err, base.MD(doc.ID), base.MD(doc.CurrentRev))
```

Comment:
```suggestion
			base.WarnfCtx(ctx, "Sync fn exception: %+v; doc %q / %q", err, base.UD(doc.ID), base.MD(doc.CurrentRev))
```

- Document IDs are user data (determined by customers/users) so should be tagged by `UD`
- Rev IDs are generated metadata so can be tagged `MD`
- Somewhat consistent logging for doc ID/rev ID pairs suggested (see logging above)